ServerCore/ServerCore.csproj

```diff
@@ -43,4 +43,15 @@
   <ItemGroup>
     <WCFMetadata Include="Connected Services" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Update="Pages\EventSpecific\FAQ.cshtml">
```

Comment:
What's up with these? Most pages don't need this sort of thing. Are you sure it's necessary?

Reply:
Odd - I don't think I touched those. It might have been something automatic?